### PR TITLE
Add Register with RHSM - ansible path

### PR DESCRIPTION
**Pull request by LI0131**
This PR fixes -> https://projects.engineering.redhat.com/browse/RHCLOUD-8022

Here's what I've got so far, some more to come.

Screen Shot 2020-07-30 at 5 17 52 PM

### CODE REVIEW

**Comment by LI0131** [comment]
Here's where we are now:

Screen Shot 2020-07-30 at 6 45 10 PM

Screen Shot 2020-07-30 at 6 44 38 PM

@AllenBW Looks like the mocks aren't really finished for the second tab, do you know if they are going to want a custom component for the expandable sections? or where the data for those will be coming from?

**Comment by LI0131** [comment]
Got the info into the accordion now to make it work with data-driven-charts...

Screen Shot 2020-07-31 at 11 52 12 AM

**Comment by LI0131** [comment]
Here are some screenshots of the finished product.

Screen Shot 2020-08-03 at 10 40 34 AM

Screen Shot 2020-08-03 at 10 40 51 AM

Screen Shot 2020-08-03 at 10 40 57 AM

Screen Shot 2020-08-03 at 10 41 05 AM

Screen Shot 2020-08-03 at 10 41 12 AM

File: src/Components/AnsibleTabs/AnsibleTabs.scss [hunk 1]
```diff
@@ -0,0 +1,102 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all.scss';
+
+section {
+    outline: none;
+}
+
+pre {
```
Comment (AllenBW) [inline src/Components/AnsibleTabs/AnsibleTabs.scss:7]: oh hey, if we're using this here as well, lets pull it outta helpers.scss and here, throw it in app.scss or something 🤔

File: src/Routes/Register/Helpers.js [hunk 2]
```diff
@@ -100,6 +101,43 @@ const schema = intl => ({
...
+                {stepTitle(intl, intl.formatMessage(messages.downloadClientPlaybook), 2)}
+                <Button variant="primary" className='pf-m-display-lg' icon={<DownloadIcon />} component='a' href='https://github.com/RedHatInsights/insights-client-role'>
+                    {intl.formatMessage(messages.downloadPlaybook)}
+                </Button>
+                <br/>
+                {stepTitle(intl, intl.formatMessage(messages.configurePlaybook), 3)}
+                <br/>
+            </React.Fragment>
+        }, {
+            component: componentTypes.TABS,
```
Comment (AllenBW) [inline src/Routes/Register/Helpers.js:120]: sexy.

**Pull request merged**